en/docs/develop/creating-artifacts/creating-an-api.md

```diff
@@ -176,7 +177,9 @@ When you create the API, an API resource is created by default. If you want to a
         </api>
         ```    
 
-### Update metadata
+<!--
+
+## Update metadata
```

Comment:
Why this is commented out?

Reply:
Since that is no longer valid: https://github.com/wso2/mi-vscode/issues/70#issuecomment-2384806474